### ISSUE TITLE
Issue on running "npm run protractor"

### ISSUE THREAD

**Issue by ezwon** [angular/angular-seed#189]
I'm trying to follow the instruction on this seed and I'm blocked by my issue with protractor. I've already done the following but still got the issue on the screenshot below:
1. Install latest JDK for Window8
2. Make sure server is started ("npm start") on another console.
3. Updated latest selenium and chromedriver.

![anuglar protractor](https://cloud.githubusercontent.com/assets/7529733/2923887/1a8b24f6-d723-11e3-82d9-256c959f468a.png)

https://github.com/angular/angular-seed/issues/176
This is already reported before but it didn't work for me. :) Any help will be really appreciated. Thanks and kudos to the AngularJS Team!
